cypress/e2e/marc-authority/marc-bibliographic.cy.js

```diff
@@ -54,6 +106,7 @@ describe('MARC -> MARC Bibliographic', () => {
         waiter: DataImport.waitLoading,
       });
       DataImport.uploadFile('marcFileForC360542.mrc', fileName);
+      cy.wait(5000);
```

Comment:
Add comment please, reason why you added waiter here

Reply:
No need for this, fixed